todolist copy 3/main.js

```diff
@@ -0,0 +1,103 @@
+"use strict";
+
+const addTaskButton = document.getElementById('addTask');
+const lineTaskInput = document.getElementById('taskInput');
+const toDoList = document.getElementById('tasklist');
+const clearing = document.getElementById('clearAll');
+const checkBoxCover = document.getElementById('clearAll');
+
+
+let tasks = JSON.parse(localStorage.getItem("tasks")) ?? [];

```

Comment:
`let tasks = JSON.parse(localStorage.getItem("tasks")) ?? [];` is refactored version of `!localStorage.tasks ? tasks = [] : tasks = JSON.parse(localStorage.getItem('tasks'));`
they are doing the same thing